client/src/layout/Footer/Footer.js

```diff
@@ -1,7 +1,272 @@
+      color: 'white',
+      '&:hover': {
+        color: 'blue',
+      },
+    },
+  },
+  '@media *min-width: 768px': {
+    paddingBottom: '0',
+  },
+});
+
+const Footer = () => (
+  <FooterContainer>
+    <FooterWrapper>
+      <Box>
+        <OurAwards>
+          <Typography variant="h3">Our Awards</Typography>
+          <Typography>
+            Ukraine is a country of brave people and countless opportunities. Here you will be accepted as a relative,
+            surrounded by care as at home and get everything done as one would do it for themselves
+          </Typography>
+          <Typography
+            sx={{ width: '250px', marginTop: '30px', paddingBottom: '40px' }}
+            component="img"
+            src="./assets/images/footer/awards.png"
+            alt="awards_mage"
+          />
+        </OurAwards>
+        <ContactsData>
+          <Typography variant="h3">Contact info</Typography>
+          <ContactsInfo>
+            <Box>
+              <SmartphoneOutlinedIcon color="white" />
+              <MUILink href="tel:+380661482997">+38 (066) 148 29 97</MUILink>
+            </Box>
+            <Box>
+              <LocationOnOutlinedIcon color="white" />
+              <MUILink
+                rel="noreferrer"
+                target="_blank"
+                href="https://www.google.com.ua/maps/place/%D0%9C%D0%B0%D1%80%D0%B8%D0%B8%D0%BD%D1%81%D0%BA%D0%B8%D0%B9+%D0%B4%D0%B2%D0%BE%D1%80%D0%B5%D1%86/@50.4438071,30.544801,15.25z/data=!4m5!3m4!1s0x40d4cfad45d00d9d:0xed19f61774e5a260!8m2!3d50.4483553!4d30.5376094?hl=ru"
+              >
+                Mariinsky palace, Kyiv, Ukraine
+              </MUILink>
+            </Box>
+            <Box>
+              <TimerOutlinedIcon />
+              <Typography>Mon - Sat 8.00 - 18.00 Sunday CLOSED</Typography>
+            </Box>
+            <SocialIcons>
+              <Box>
+                <FacebookRoundedIcon />
+              </Box>
+              <Box>
+                <TwitterIcon />
+              </Box>
+              <Box>
+                <YouTubeIcon />
+              </Box>
+              <Box>
+                <PinterestIcon />
+              </Box>
+              <Box>
+                <InstagramIcon />
+              </Box>
+            </SocialIcons>
+          </ContactsInfo>
+        </ContactsData>
+        <Box>
+          <Typography sx={{ color: 'white' }} variant="h3">
+            Recent Trips
+          </Typography>
+          <RecentTrips>
+            {footerGallery.map((el) => (
+              <Link to="/catalogue">
+                <Typography component="img" src={el.imagePath} alt={el.altDescription} />
+              </Link>
+            ))}
+          </RecentTrips>
+        </Box>
+      </Box>
+    </FooterWrapper>
+    <FooterNavigation>
+      <Box>
+        <Typography component="nav">
+          <NavigationList>
+            {navigationMenu.map((el) => (
+              <Typography key={el.name} component="li">
```

Comment:
just use a li and wrap it with a ul.  no need for typography here

Reply:
change Typography li to li component. NavigationList it`s my UL component here. Solved.